Include/internal/pycore_freelist.h

```diff
@@ -71,6 +71,14 @@ struct _Py_dict_freelist {
 #ifdef WITH_FREELISTS
     /* Dictionary reuse scheme to save calls to malloc and free */
     PyDictObject *free_list[PyDict_MAXFREELIST];
+    int numfree;
+    int keys_numfree;
```

Comment:
I think we need only one of `numfree` or `keys_numfree` in each struct.

Reply:
nah copy and paste accident :)